packages/grid/_modules_/grid/components/Pagination.tsx

```diff
@@ -10,12 +10,12 @@ import { isMuiV5 } from '../utils';
 // Used to hide the drop down select from the TablePaginagion
```

Comment:
this comment is not clear. Which drop down? Why...

Reply:
Just a note, I know it wasn't changed :)

Reply:
Should be this one:

![image](https://user-images.githubusercontent.com/4512430/105505171-ec251580-5cc8-11eb-8203-58c6df572568.png)

Maybe the following is better:

```suggestion
// Used to hide the Rows per page selector
```

Reply:
```suggestion
// Used to hide the Rows per page selector on small devices
```

This may be even better :)